shell/basic.sh

```diff
@@ -35,13 +35,13 @@ read fruit
 # Check the entered fruit using a case statement
 case $fruit in
   "apple")
-    echo "You entered apple."
+    echo "You entered apple 🍎"
```

Comment:
There are still change from other task included to this PR, could you please double check? Thanks!